gatsby-config.js

```diff
@@ -170,6 +172,23 @@ const getPlugins = () => {
         outputConfigFile: `${__dirname}/nginx.out.conf`,
       },
     },
+    {
+      resolve: "./plugins/gatsby-plugin-openapi",
+      options: {
+        name: "openapi",
+        resolve: async () => {
+          try {
+            const response = await axios.get(
+              "https://raw.githubusercontent.com/getsentry/sentry-api-schema/68bb79acfbbee062bd8d2f71ee3a07d43dc934c9/openapi-derefed.json"
```

Comment:
is this temporary? should it be master? does it need updated regularly? at the very least its probably worht a comment explaining the url

Reply:
@dcramer I wrote a small note in the Objective addressing this. Currently, the plan is to fix it to a specific SHA from `getsentry/sentry-api-schema` and update it manually. Eventually we can have a bot/github action do it automatically.